admin/services/data-access/pupil-access-arrangements.data.service.js

```diff
@@ -129,4 +129,32 @@ pupilAccessArrangementsDataService.sqFindPupilsWithAccessArrangements = async (d
   return sqlService.query(sql, params)
 }
 
+pupilAccessArrangementsDataService.sqlFindAccessArrangementsByUrlSlug = async (urlSlug) => {
+  const params = [
+    {
+      name: 'urlSlug',
+      value: urlSlug,
+      type: TYPES.NVarChar
+    }
+  ]
+  const sql =
+    `SELECT
+    p.urlSlug,
+    p.foreName,
+    p.lastName,
+    paa.inputAssistanceInformation,
+    paa.questionReaderOtherInformation,
+    aa.code as accessArrangementCode,
+    qrr.code as questionReaderReasonCode 
+    FROM ${sqlService.adminSchema}.pupilAccessArrangements paa
+    INNER JOIN ${sqlService.adminSchema}.pupil p
+      ON p.id = paa.pupil_id
+    INNER JOIN ${sqlService.adminSchema}.accessArrangements aa
```

Comment:
is there always going to be a matching record between `pupilAccessArrangements` and `accessArrangements`?

if not, an inner join will omit records without a match

Reply:
There should always be at least 1 matching record of `accessArrangements` for  `pupilAccessArrangements ` since this is being called from the edit controller method.

Reply:
👍